converters/convert_to_scripture_burrito.py

```diff
@@ -0,0 +1,82 @@
+import os
+import json
+
+
+# These are not in the standard format.
+ALIGNMENT_EXCEPTIONS = [
+    "WLC-NET-manual.json",
+    "WLC-CSBE-manual.json",
+    "WLC-YLT-manual.json",
+    "WLC-SGS-manual.json",
+    "NA27-SGS-manual.json",
+    "NA27-HSB-manual.json",
+    "NA27-CUVMP-manual.json",
+]
+
+
+def find_alignment_file_paths_for_conversion():
+    alignment_file_paths = []
+    for root, dirs, files in os.walk("data/alignments"):
+        for file in files:
+            if file.endswith("-manual.json") and file not in ALIGNMENT_EXCEPTIONS:
+                alignment_file_paths.append(os.path.join(root, file))
+    return alignment_file_paths
+
+
```

Comment:
@themikejr no problems with this, but at some point you might want to look at bible_alignments/config.py for dealing with the various pieces of alignment files.